### DISCUSSION TITLE
Please add 'Next Month' as an period option, I often use Last Month, Current Month and the manually have to do 'Next Month'

### DISCUSSION THREAD

**Discussion by diepeterpan** [moneymanagerex/moneymanagerex#5622] [Ideas]
Please add 'Next Month' as a period option in reports, I often use 'Last Month', and 'Current Month' and then manually have to do 'Next Month' to view transactions/expenses captured for the next month.

(https://github.com/moneymanagerex/moneymanagerex/blob/master/src/reports/mmDateRange.cpp)

**Comment by whalley** [comment]
You could use the 'Cash Flow' / 'Transactions report' and select Future Months as 1

**Reply by diepeterpan** [reply]
> You could use the 'Cash Flow' / 'Transactions report' and select Future Months as 1

I apology for not being precise, I wrote a custom report where I need the option in the dropdown.  Can one add something to the drop-down on a custom report? (any help on how)

![image](https://user-images.githubusercontent.com/15066052/218278711-9dae3edf-d57c-4b0b-a66c-e776a7b9c7f2.png)
